Arrowgene.Ddon.GameServer/Handler/InstanceGetEnemySetListHandler.cs

```diff
@@ -43,10 +51,59 @@ public override void Handle(GameClient client, StructurePacket<C2SInstanceGetEne
                     PositionIndex = i,
                     EnemyInfo = spawn.asCDataStageLayoutEnemyPresetEnemyInfoClient()
                 };
-                response.EnemyList.Add(enemy);
+                // Get spawn time from the enemy object
+                string spawnTime = spawn.SpawnTime;
+                long startMilliseconds, endMilliseconds;
+                ConvertSpawnTimeToMilliseconds(spawnTime, out startMilliseconds, out endMilliseconds);
```

Comment:
The conversion shouldn't be done every time the handler is called but rather when the file is being loaded. Doing it like this means the same calculation is done every time the handler is called, instead of just once on load.